### ISSUE TITLE
Accent seems not to be working in range slider

### ISSUE THREAD

**Issue by PabloRuizCuevas** [skeletonlabs/skeleton#679]
### Current Behavior

I expect the part of the color of the slider to change when changing accent in the docs,. 

### Steps To Reproduce

got to: https://www.skeleton.dev/components/range-sliders
Change accent color, in firefox or chrome nothing changes.

**Comment by endigo9740** [comment]
@PabloRuizCuevas are you supplying the color value via the `accent` prop? Make sure you're using the Tailwind provided utility class format for this:
https://tailwindcss.com/docs/accent-color

If you're using a Skeleton theme color you're want to use something like `accent-[color]-500`, where color is `primary`, `accent`, etc.

Finally, given that we set this color globally via the `forms.css` stylesheet, it may be a scenario where you have to set `!` (important) to override the defaults. Try using something like:

`accent="!accent-tertiary"`

Testing on our documentation it looks like our interactive example was having the same issue. Setting "important" fixed this.

![Screen Shot 2022-12-13 at 9 00 01 AM](https://user-images.githubusercontent.com/1509726/207368015-6bb0beaf-f018-4e23-8960-372b9da6583a.png)

![Screen Shot 2022-12-13 at 9 00 18 AM](https://user-images.githubusercontent.com/1509726/207368105-d5a3d9ec-4601-41a1-9488-f0883b5cf0bb.png)

**Comment by endigo9740** [comment]
I'm going to go ahead and close this as I suspect this will resolve your issue, but let me know if it does not and I'll gladly reopen!

**Comment by PabloRuizCuevas** [comment]
@endigo9740 Yes, that's great, many thanks!

**Comment by MrAmericanMike** [comment]
> 

Would it be possible to update the documentation accordingly?

https://www.skeleton.dev/components/range-sliders

The list shows the accent colors names without the ! in front there.

PS: (I would but not sure where that page lives in the repository)

**Comment by endigo9740** [comment]
@MrAmericanMike I might recommend avoiding posting in old, closed messages. A new post referencing this would be fine. But per your request, this has been resolved in the next iteration of the docs dropping later this week:
https://dev.skeleton.dev/components/range-sliders

Instead of `!` we've opted to show both the light and dark mode colors in the Props tab, which is another (probably better) alternative.
